### PR TITLE
Correct multitask data loading issue.

### PR DESCRIPTION
**Pull request by rbharath**
Updates once complete.

### CODE REVIEW

**Comment by rbharath** [comment]
Changes Made:

-) Fixed critical bug in loading of multitask dataset. (A `np.reshape` was jumbling data between different targets, leading to nonsensical results). Added test (`deepchem/datasets/tests/test_load.py`) to maintain load correctness for future changes.
-) Changed order of arguments to `Dataset.from_numpy()` to allow for easy load of datasets (calling `Dataset.from_numpy(X, y)` will now work, which allows for easy inter-op with `sklearn.datasets`
-) Added stub `Metric` test to `deepchem.metrics.tests`. Need to add more tests to satisfy #168.
-) Added `sklearn` tests to verify that models learn generalizable function in `deepchem.models.tests.test_generalize`. Need to add `keras` and `tensorflow` tests before closing #164 though.
-) Updated models in `examples`. Still need more work before models are stable though (MUV models provide `NaN`, possibly due to bad data split).
